egress/src/lib.rs

```diff
@@ -131,6 +133,11 @@ pub async fn run(pool: PgConnectionPool, cancel_token: CancellationToken) {
             get(timeslice_handler),
         )
         .route("/latest", get(latest_handler))
+        .route(
+            "/reports/idf/station",
+            get(idf_station_availability_handler),
+        )
+        .route("/reports/idf/station/:station_id", get(idf_station_handler))
```

Comment:
I think nested routers would make this neater, and would let us move the routing for idf into the idf module.

Reply:
Good point!

Reply:
Done!